File: src/ERP/acme.atena.domain/DTO/Util/EnderecoPessoa.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Util
{
    public abstract class EnderecoPessoa : AbstractEntity
    {
        public string Numero { get; set; }
        public string Complemento { get; set; }
        public string Latitude { get; set; }
        public string Longitude { get; set; }
        public Guid EnderecoId { get; set; }

        public virtual Endereco Endereco { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/util/endereco-pessoa.ts
import { Endereco } from "./endereco";
import { AbstractEntity } from "../abstract-entity";

export abstract class EnderecoPessoa extends AbstractEntity {
  public numero: string;
  public complemento: string;
  public latitude: string;
  public longitude: string;
  public enderecoId: string;

  public endereco: Endereco= new Endereco();
}
